src/main/java/com/andersen/config/JpaConfig.java

```diff
@@ -0,0 +1,62 @@
+package com.andersen.config;
+
+import lombok.extern.slf4j.Slf4j;
+import org.hibernate.cfg.Environment;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.JpaVendorAdapter;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+import java.util.Properties;
+
+@Import(DataSourceCfg.class)
+@Configuration
+@ComponentScan(basePackages = {"com.andersen"})
+@EnableTransactionManagement
+@Slf4j
```

Comment:
not needed

Reply:
without @ComponentScan and @EnableTransactionManagement, I get an error

Reply:
@chingiz-mazhitov this comment was just to Slf4j
so, return Configuration please